src/main.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+import { NFC } from 'nfc-pcsc';
+import EthTx from 'ethereumjs-tx';
+import LuksoNfcChip from './reader/LuksoNfcChip';
+import Tlv from './tlv/Tlv';
+
+const nfc = new NFC();
+
+nfc.on('reader', async reader => {
+  console.log(`${reader.reader.name}  device attached`);
+  reader.aid = '654E6F7465734170706C6574';
```

Comment:
What is this hard coded value?

Reply:
```AID``` is Application IDentifier. There are serveral applicatons in chip. We must specify which appliation we want to interact by issue a standard ```Slect AID``` APDU Commond.